src/components/pages/career-details/career-details-structured-data.tsx

```diff
@@ -20,48 +25,57 @@ const concatDescription = (position: SyPersonioJob): string => {
   );
 };
 
+/**
+ * Create and write structured data based on https://schema.org/JobPosting
+ *
+ * See the Google guide for additional insights:
+ * https://developers.google.com/search/docs/advanced/structured-data/job-posting#structured-data-type-definitions
+ */
 export const CareerDetailsStructuredData = ({
   position,
 }: CareerDetailsStructuredDataProps) => {
-  const employmentType = convertScheduleToEmploymentType(position.schedule);
-
-  const data = {
-    '@context': 'https://schema.org/',
+  const structuredData: WithContext<JobPosting> = {
+    '@context': 'https://schema.org',
     '@type': 'JobPosting',
     title: position.name,
     datePosted: position.createdAt,
     description: concatDescription(position),
     identifier: {
       '@type': 'PropertyValue',
       name: 'Satellytes Digital Consulting GmbH',
-      value: '338504-41230',
+      value: position.jobId,
```

Comment:
this is crucial, it's the id of the job we have to provide here

Reply:
ah, okay, I was not sure what the identifier does, but that makes sense 👍